### DISCUSSION TITLE
webview support high DPI?

### DISCUSSION THREAD

**Discussion by Jzhenli** [beeware/toga#3028] [Q&A]
Hi, I am using Win11 with toga webview and I have a high DPI monitor with 150% scaling. The problem is that the whole app looks blurry. Not sure if toga webview support high DPI?

**Comment by freakboy3742** [comment]
We're not doing anything toe explicitly enable or disable HiDPI handling on a per-widget basis. If the Winforms API requires a specific setting to enable DPI mode, we're not aware of it.

**Reply by mhsmith** [reply]
This isn't specific to the webview, but "the whole app looks blurry" is a known problem which we recently fixed (#2155), but have not yet released.

**Reply by Jzhenli** [reply]
Yes, that's exactly what I mean. I'm looking forward to the new version can fix this issue. @mhsmith

**Reply by Jzhenli** [reply]
> This isn't specific to the webview, but "the whole app looks blurry" is a known problem which we recently fixed (#2155), but have not yet released.

Does this feature have been merged in V0.4.9? @mhsmith

**Reply by mhsmith** [reply]
No, it will be released as part of version 0.5.0, probably within the next month.

**Reply by Jzhenli** [reply]
> No, it will be released as part of version 0.5.0, probably within the next month.

@mhsmith I tried V0.5.0 and found some issues. My laptop system is Windows 11 with a screen resolution of 2240x1400 and scaling at 150%. There is an external display connected to the laptop, with a resolution of 2560x1440 and scaling at 100%. When my Toga application is displayed on the external display, it works normally. However, when I drag and drop it onto the laptop's own screen, there will be a large shadow on the Toga application. As shown in the following picture
![image](https://github.com/user-attachments/assets/f243f7ec-f813-4950-8a30-e2b1c671fdcf)

**Reply by mhsmith** [reply]
It looks like this glitch is happening inside a WebView. Please make an example app reproducing this with the smallest possible amount of code, and the simplest possible HTML file, and post the Python and HTML source to a new issue.

**Reply by Jzhenli** [reply]
ok, I have open issue [3264](https://github.com/beeware/toga/issues/3264)  @mhsmith
